src/main/java/hwr/oop/riddler/model/component/Cell.java

```diff
@@ -5,9 +5,12 @@
 
 public class Cell {
     public int value;
-    public Set<Integer> possibles = new HashSet<>(Set.of(1, 2, 3, 4, 5, 6, 7, 8, 9));
+    public Set<Integer> possibles;
     
-    Cell(int value) {
+    public Cell(int value) {
         this.value = value;
+        if (value == 0) {
```

Comment:
Constructor overloading instead?

public Cell(int value) {
    this.value = value;
}

public Cell() {
    possibles = new HashSet<>(Set.of(1, 2, 3, 4, 5, 6, 7, 8, 9));
}